.travis.yml

```diff
@@ -26,7 +27,11 @@ before_install:
   - git clean -qxdff
 
 script:
-  - brew test-bot
+  - if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
```

Comment:
Done.

Thought, I feel like we should sort out these logics (but I have no clue what purpose for each branches are) https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/cmd/test-bot.rb#L302-L355